<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Визуализатор графиков</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="6">
     <widget class="QCheckBox" name="cbLegend">
      <property name="text">
       <string>Легенда</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxXmin">
      <property name="minimum">
       <double>-100000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxXmax">
      <property name="minimum">
       <double>-100000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxYmin">
      <property name="minimum">
       <double>-100000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxYmax">
      <property name="minimum">
       <double>-100000000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="separator"/>
    <addaction name="clearAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="previewAction"/>
    <addaction name="separator"/>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>Chart</string>
    </property>
    <addaction name="titleChartAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuChart"/>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Open file...</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="action_function">
   <property name="text">
    <string>Set expression...</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="previewAction">
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="titleChartAction">
   <property name="text">
    <string>Title...</string>
   </property>
  </action>
  <action name="actionSet_parser">
   <property name="text">
    <string>Set parser...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
